src/linear_algebra/dimension.lean

```diff
@@ -462,10 +462,20 @@ begin
     exact le_antisymm w₁ w₂, }
 end
 
+/-- Given two basis indexed by `ι` and `ι'` of an `R`-module, where `R` satisfies the invariant
+basis number property, an equiv `ι ≃ ι' `. -/
+def equiv_of_basis (v : basis ι R M) (v' : basis ι' R M) : ι ≃ ι' :=
```

Comment:
Strangely, I didn't find this anywhere.

Reply:
Perhaps this should be called [`basis.index_equiv`](https://leanprover-community.github.io/mathlib_docs/linear_algebra/free_module_pid.html#basis.index_equiv) since it's apparently a good generalization of that one. (I don't think `invariant_basis_number` was defined back then.)